NightscoutServiceKit/Extensions/StoredSettings.swift

```diff
@@ -19,9 +19,10 @@ extension StoredSettings {
 
         var nightscoutPreMealTargetRange: ClosedRange<Double>?
         if let preMealTargetRange = preMealTargetRange {
-            nightscoutPreMealTargetRange = ClosedRange(uncheckedBounds: (
-                lower: preMealTargetRange.minValue,
-                upper: preMealTargetRange.maxValue))
+            nightscoutPreMealTargetRange = ClosedRange(
+                uncheckedBounds: (
+                    lower: preMealTargetRange.doubleRange(for: bloodGlucoseUnit).minValue,
```

Comment:
[nit] Consider pulling out `preMealTargetRange.doubleRange(for: bloodGlucoseUnit)` into a local variable so the work isn't done twice.